src/core/monitor.tf

```diff
@@ -7,3 +7,8 @@ data "azurerm_monitor_action_group" "error_action_group" {
   name                = "${var.prefix}${var.env_short}error"
   resource_group_name = azurerm_resource_group.rg_common.name
 }
+
+data "azurerm_monitor_action_group" "io_com_action_group" {
+  name                = "io-p-com-error-ag-01"
+  resource_group_name = "io-p-itn-msgs-rg-01"
+}
```

Comment:
since this data is not longer used, it can be removed

Reply:
https://github.com/pagopa/io-infra/pull/1164/commits/7b729bdb44e7bda346bb6ebcfd00a1f9a4839a0d